src/mg_biome.cpp

```diff
@@ -83,6 +83,40 @@ void BiomeManager::clear()
 	m_objects.resize(1);
 }
 
+
+Biome *BiomeManager::getBiomeAtPos(v3s16 pos)
+{
+	float heat =
+		NoisePerlin2D(&m_params->np_heat,       pos.X, pos.Z, m_params->seed) +
+		NoisePerlin2D(&m_params->np_heat_blend, pos.X, pos.Z, m_params->seed);
+	float humidity =
+		NoisePerlin2D(&m_params->np_humidity,       pos.X, pos.Z, m_params->seed) +
+		NoisePerlin2D(&m_params->np_humidity_blend, pos.X, pos.Z, m_params->seed);
+	printf("heat %f\n", heat);
+	printf("humidity %f\n", humidity);
+
+	Biome *b, *biome_closest = NULL;
```

Comment:
Can you use nullptr ?

Reply:
That might be better, i was just copying code from later in the file.

Reply:
go 🗡

Reply:
would be better to define b in side the loop, imo

Reply:
Ok.

Reply:
Both done.